crypto/erc2335/erc2335.go

```diff
@@ -0,0 +1,83 @@
+package erc2335
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	keystorev4 "github.com/wealdtech/go-eth2-wallet-encryptor-keystorev4"
+)
+
+type Erc2335KeyStore struct {
+	Crypto      map[string]interface{} `json:"crypto"`
```

Comment:
Some doc for this field would be nice like what does this map hold (as type ` map[string]interface{}` does not tell much to the reader)